.github/scripts/build-and-run-sanitizers.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Copyright (c) 2019 Kevin R Croft <krcroft@gmail.com>
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# This simple script is used to build, test, and archive the log output from
+# various sanitizer-builds.
+#
+# Usage:
+#   ./build-and-run-sanitizers.sh COMPILER VERSION SANITIZER [SANITIZER [...]]"
+#
+# Examples:
+#   ./build-and-run-sanitizers.sh clang 8 msan usan
+#   ./build-and-run-sanitizers.sh gcc 9 asan uasan usan tsan
+#
+set -euo pipefail
+
+# Configurable compression settings:
+ZSTD_OPTS="--long=31 -19 -T0 -v"
+
+# Check the arguments
+if [[ "$#" -lt 3 ]]; then
+  echo "Usage: $0 COMPILER VERSION SANITIZER [SANITIZER [...]]"
```

Comment:
Will update. (I thought we were using spaces for stuff that won't be merged upstream) Should we use tabs across the board?

Reply:
Frankly: I don't know. Personally, I was trying to follow a rule to use tabs everywhere, except Python (because using PEP-8 recommends spaces for new code and Python developers don't expect tabs) and .yml files (because GitHub templates and examples do not use tabs), but maybe that's too restrictive.

Reply:
I like this approach. Stick to tabs where existing code uses it (DOSBox-core and Bash); but use current-recommended-practices for anything else.

Reply:
Fixed in dbcaee82